download.php

```diff
@@ -75,7 +75,8 @@
         if ($bean_name == 'aCase') {
             $bean_name = 'Case';
         }
-        if (!file_exists('modules/' . $module . '/' . $bean_name . '.php')) {
+        global $beanFiles;
+        if (!file_exists($beanFiles[$bean_name])){
```

Comment:
Hi @sweettbug3,

thank you for your PR.

Could you also add a validation to check if the `$bean_name` within `$beanFiles`.

Maybe something like

```
global $beanFiles;
$beanFileName = $beanFiles[$bean_name] ?? '';
if ($beanFileName === '' || !file_exists($beanFileName)){
```